dll/win32/shell32/utils.cpp

```diff
@@ -540,6 +540,123 @@ LargeIntegerToString(
                          pNumberFormat, dwNumberFlags);
 }
 
+/*************************************************************************
+ *  CopyStreamUI [SHELL32.726]
+ *
+ * Copy a stream to another stream with optional progress display.
+ */
+EXTERN_C
+HRESULT WINAPI
+CopyStreamUI(
```

Comment:
Do you know an application that uses this function?

Reply:
`shell32` uses this function in `CLocalCopyHelper::Download`, `CLocalCopyHelper::Upload`, and `CopyStreamToFile`.

Reply:
`CopyStreamToFile` is referred by `CFSDropTarget::_CreatePackage`.